### DISCUSSION TITLE
Mojolicious::Guides::Growing ... how about the final directory structure contents tar file?

### DISCUSSION THREAD

**Discussion by westmj** [mojolicious/mojo#1685] [Q&A] [answered]
Mojolicious::Guides::Growing ( https://metacpan.org/pod/Mojolicious::Guides::Growing ) seems like a wonderful tutorial to work out, but I am getting lost. I sure would like a .tar file or such I could decompress on my own machine, to play with the final product.  Is that available anywhere?  Basically, the contents of:

myapp
|- script
|  +- my_app
|- lib
|  |- MyApp.pm
|  +- MyApp
|     |- Controller
|     |  +- Login.pm
|     +- Model
|        +- Users.pm
|- t
|  +- login.t
+- templates
   |- layouts
   |  +- default.html.ep
   +- login
      |- index.html.ep
      +- protected.html.ep

Thanks in advance! Alternatively, some full, worked-out well-structured applications that make MVC obvious. I work better off working examples I can tinker with, than theory.

**Comment by kraih** [comment, marked as answer]
Good point. It now ships with Mojolicious as an actual example. 😁 https://github.com/mojolicious/mojo/compare/0a4f64072bc4...2a20744ef485
